src/WebJobs.Script/Workers/ProcessManagement/EmptyProcessRegistry.cs

```diff
@@ -8,5 +8,9 @@ namespace Microsoft.Azure.WebJobs.Script.Workers
     internal class EmptyProcessRegistry : IProcessRegistry
     {
         public bool Register(Process process) => true;
+
+        public void Close()
```

Comment:
Just curious, why do we need an empty process registry?

Reply:
saw usage of EmptyProcessRegistry in Tests projects.